h/api/storage.py

```diff
@@ -252,18 +195,3 @@ def expand_uri(request, uri):
             return [uri]
 
     return [docuri.uri for docuri in docuris]
-
-
-def _prepare(request, annotation):
-    """Prepare the given annotation for storage."""
-    fetcher = partial(fetch_annotation, request, _postgres=False)
-    transform.prepare(annotation, fetcher)
```

Comment:
I'm not 100% sure but I think this means the entirety of `h.api.transform` can go now, too.